### DISCUSSION TITLE
Documentation improvement

### DISCUSSION THREAD

**Discussion by bordenc** [Kozea/Radicale#1388] [Ideas]
Similar to other issues, I'm struggling to follow Radicale's documentation. I'm willing to invest some time to improving it and requesting a pull, but I'm stuck in a chicken-and-egg problem. I don't know enough about Radicale to propose documentation improvements, yet I can't figure out Radicale because of the documentation.

What's the best way to collaborate with the devs and ask questions about where in the code I can find configuration options and/or get the necessary background to hack away at the documentation? Should I post my questions with citations here? Is there a way I could share a copy of DOCUMENTATION.md marked up with all of my questions and feedback?

**Comment by SuperMeepEnby** [comment]
Hi !

Might be late but better late than never I guess.

My opinion about that is that everyone that had trouble with the documentation should just point out where and what are they willing to accomplish so we can improve it.

I don't see any other way of doing it.

**Comment by bordenc** [comment]
What's the easiest way for you to accept suggestions? Though this thread? Diff/patch requests? Wiki edits?

I have so far received (fairly) good feedback when/where I can post edit suggestions with commentary on what I found difficult about the original documentation. That way it's not "edits for the sake of edits."

**Comment by SuperMeepEnby** [comment]
I'm not a Radicale main dev but I'm willing to take a look at the code and improve it. I may have some time to improve documentation first. You can just share here the points that gave you hard time and maybe explain what you were trying to accomplish and why the documentation wasn't helpful/what you would have expected ? :)

I honestly don't know I'm just popping from nowhere willing to contribute x)

Edit: if you have already tweaked some parts of the documentation you may be able to do a pull request I guess, I haven't read the contributing guidelines yet but I'm sure you can :)
